code/modules/admin/verbs/one_click_antag.dm

```diff
@@ -477,13 +477,13 @@
 			//Open the Armory doors
 			if(ertemplate.opendoors)
 				for(var/obj/machinery/door/poddoor/ert/door in GLOB.airlocks)
-					door.open()
+					INVOKE_ASYNC(door, /obj/machinery/door/poddoor.proc/open)
 					CHECK_TICK
```

Comment:
What's the point of this if you're going to meme with invoke async

Reply:
I do not understand the point of your comment

Reply:
What is the point of CHECK_TICK here

Reply:
Ah, I thought you were commenting on a different line